spec/requests/transition_checker_email_subscription_spec.rb

```diff
@@ -66,27 +66,16 @@
     end
 
     context "when the user doesn't have a high enough level of authentication" do
-      let(:status) { 204 }
       let(:session_identifier) { placeholder_govuk_account_session(level_of_authentication: "level-1") }
 
-      it "checks if there is a subscription" do
+      it "returns a 403 and the required level" do
```

Comment:
Do we need to updta the [docs](https://github.com/alphagov/account-api/blob/main/docs/api.md#get-apitransition-checker-email-subscription) too?

Reply:
This endpoint actually uses `level0` - so I'll remove this test (there's no such thing as `level-1`).  But the docs for the other endpoint do, yes

Reply:
Actually... I guess it's not impossible we decide to bump the permission for checking up to `level1`.  So I'll leave the check in place and just update the docs.